README.md

```diff
@@ -0,0 +1,74 @@
+# ReactJS Boilerplate
+
+## Roadmap
+
+Our roadmap for the ReactJS Boilerplate is where you can learn about what features we're working on.
+
+### Development environment
+
+- Build tool
+  - [x] Vite
+    - Optimisation of Minify code in production mode
+- Coding Standards and Guidelines (Statically analyzes our codes)
+  - [ ] ESLint with our rules
+  - [ ] Prettier
+- Testing framework
```

Comment:
@jinmayamashita I think we should add end-to-end testing too

Reply:
@mvn-luanvu-hn Indeed. 👍 Do you have any tools you would recommend? 

e.g TestCafe, Playwright

Reply:
@jinmayamashita I hear about cypress a lot. Maybe we can consider it.
But currently it does not support Safari
<img width="1351" alt="image" src="https://user-images.githubusercontent.com/23349714/158778384-f4621554-2a6e-4d72-b20b-0e0f31727fa2.png">

Reply:
@jinmayamashita @mvn-luanvu-hn
Storybook uses playwright for its[ interaction testing](https://storybook.js.org/blog/interaction-testing-with-storybook/), so we could experiment with that. I've included a simple test in the [vue boilerplate](https://github.com/monstar-lab-oss/vuejs-boilerplate/blob/base/_storybook/stories/flows/LoginFlow.stories.tsx).

Reply:
@mvn-luanvu-hn @yannde 
Added https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/27/commits/d0a398e9197834f5f50a7f21d71f6b36a5f5c3ad

I have not used Cypress yet and would like to try it out. and we uses Storybook in boilerplate, so I would like to try working with playwright. This looks like another good PR.

Reply:
@jinmayamashita I see in our project another team is using this tool
https://codecept.io/basics/#architecture
It can work well with many other tools

Reply:
@mvn-luanvu-hn Nice! This may also work with playwright :+1: Do you have a sandbox?